ui/src/components/filter/components/Label.vue

```diff
@@ -26,7 +26,9 @@
         const {value, label, comparator} = props.option;
 
         if (!value.length) return;
-
+        if (label === "labels") {
```

Comment:
This change is not expected, I would say, as the result is this view:

![image](https://github.com/user-attachments/assets/05f9b6da-4255-4db3-be45-5f348427615d)

What was it you were trying to achieve here @aeSouid?